examples/host_commands.h

```diff
@@ -169,6 +169,8 @@ struct ec_request_reset_target {
 
 /* Reset the target device. */
 #define EC_PRV_CMD_HOTH_RESET_TARGET 0x0012
+#define EC_PRV_CMD_HOTH_PAYLOAD_UPDATE 0x0005
```

Comment:
nit: Include a blank line to separate from the reset-target above -- or maybe a comment `/* Payload update commands. */`

Reply:
Removed payload-related constants from this change (did not intend to add them)